OrbitService/ProcessListTest.cpp

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+//
+
+#include <unistd.h>
+
+#include "OrbitBase/Logging.h"
+#include "ProcessList.h"
+#include "gtest/gtest.h"
+
+namespace orbit_service {
+
+TEST(ProcessList, ProcessList) {
+  ProcessList process_list;
+  const auto result1 = process_list.Refresh();
+  ASSERT_TRUE(result1) << result1.error().message();
```

Comment:
I don't mind to write it differently. In this particular case I copied the code from somewhere else because of the stream-operator which I hadn't used yet in gtest.

Reply:
I like how it is done currently. (Also, the stream operator is independent from the bool conversion. If i understood it right, the stream operator is only for producing better error messages)

Reply:
Yes, I only mentioned the streaming operator to explain that I copied that code from somewhere else where the streaming operator was used.